packages/dd-trace/src/opentracing/span.js

```diff
@@ -99,7 +98,10 @@ class DatadogSpan {
 
       unfinishedRegistry.register(this, operationName, this)
     }
-    spanleak.addSpan(this, operationName)
+
+    if (tracer?._config?.spanLeakDebug > 0) {
```

Comment:
Please add a TODO entry to replace the tests that cause the tracer from being nullish.
The tracer should AFAIK never be nullish effectively.

Reply:
Added a TODO, is there any other remaining concern?